_episodes/01-numpy.md

```diff
@@ -222,12 +222,9 @@ need for each program.
 
 > ## Scientists Dislike Typing
 >
-> We will always use the syntax `import numpy` to import NumPy.
-> However, in order to save typing, it is
-> [often suggested](http://www.scipy.org/getting-started.html#an-example-script)
-> to make a shortcut like so: `import numpy as np`.
-> If you ever see Python code online using a NumPy function with `np`
-> (for example, `np.loadtxt(...)`), it's because they've used this shortcut.
+> We will always use the `import numpy` syntax to import NumPy.
+> However, to save typing, shortcuts such as `import numpy as np` are frequently used.
```

Comment:
I guess the "saving" comes later as when using all the NumPy functions you just need `np.*`  rather than `numpy.*` which can makes some lines of could that call multiple functions a lot shorter.  But you are right, that is not evident given the ways things are currently described.

Reply:
I think the callout works better now, and I've tried to explain why people do it, and the possible consequences of this - so that people will hopefully understand why it matters